### PR TITLE
Ping on Windows - Set codepage to English

### PR DESCRIPTION
**Pull request by shigeya-dd**
### What does this PR do?

On non-US Windows, output of the ping.exe is localized.
It breaks the ping check and results in error.
To avoid this, I added the code to set the code page to English.
chcp and code page: [docs](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/chcp)

### Motivation

Feedback and support tickets from users.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

File: ping/datadog_checks/ping/ping.py
Comment (ChristineTChen) [inline ping/datadog_checks/ping/ping.py:29]:
```suggestion
    def _exec_ping(self, timeout, target_host):
        precmd = []
        if platform.system() == "Windows":  # pragma: nocover
            precmd = ["cmd", "/c", "chcp 437 &"]  # Set code page to English for non-US Windows
```

Instead of setting precmd to `[]` at every logic branch, perhaps you can just set it at the beginning of the function and just update for the Windows branch. This reduces repetitive code.

File: ping/datadog_checks/ping/ping.py
Comment (ChristineTChen) [inline ping/datadog_checks/ping/ping.py:36]:
```suggestion
```

we can remove this here if we set precmd at the top of the function

File: ping/datadog_checks/ping/ping.py
Comment (ChristineTChen) [inline ping/datadog_checks/ping/ping.py:43]:
```suggestion
```
same here

**Review by ChristineTChen — CHANGES_REQUESTED** [review]
@shigeya-dd ,

thanks for the PR. I left a comment just to clean up the code a little bit. I'll still wait on merging until the maintainer has had a chance to review.

**Comment by shigeya-dd** [comment]
Thanks @ChristineTChen ! I cleaned up the codes.

**Review by ChristineTChen — APPROVED** [review]
LGTM! Pinging @jstanton617 as the maintainer for a look as well

**Comment by ChristineTChen** [comment]
Hi @jstanton617 , quick follow up on my previous ping.

**Review by jstanton617 — APPROVED** [review]
LGTM!

**Pull request merged**
